src/content/resources/index.mdx

```diff
@@ -27,6 +27,7 @@ To see a comprehensive overview of OASIS+ Apparent Contractor Awardees across th
 | [Symphony Task Order System - How to use the portal](https://industrysupport.apexlogic.com/support/solutions/articles/35000224610-how-to-use-the-oasis-symphony-portal) | Provides instructions for getting started with Symphony. |
 | [Symphony Task Order System - How to build an offer](https://industrysupport.apexlogic.com/support/solutions/articles/35000222331-how-to-build-your-offer-for-oasis-) | Provides instructions for building an offer in Symphony. |
 | [Frequently Asked Questions](https://industrysupport.apexlogic.com/support/solutions/articles/35000223291-oasis-faq) | Frequently asked questions about Symphony. |
+| [OASIS+ Systems, Tools, and Registration Sellers Guide](https://github.com/GSA/oasis-plus/blob/smteague-patch-2/public/files/OASIS%2B_Systems%2C%20Tools%2C%20and%20Registration_Sellers%20Guide_%20Sep%202024%20(508).pdf) | A guide to the various registrations and tools required to manage your contract to help your company be in compliance with the OASIS+ program requirements. |
```

Comment:
@smteague when we uploaded file to public folder, it is not external link, we can't use the github branch link as the link, you should remove http part all the way util you reached the "/files/", like this
.  when you `[OASIS+ Systems, Tools, and Registration Sellers Guide](/files/OASIS%2B_Systems%2C%20Tools%2C%20and%20Registration_Sellers%20Guide_%20Sep%202024%20(508).pdf)`, there is reference in the [how to doc](https://docs.google.com/document/d/1UgbWZZrSoaZpejDhbOkbiZLYfALajJDnKIjbZDPizRM/edit), you can give a try, and see if it resolve your issue, please ping me when you need help, I can help you correct it